### PR TITLE
Docu

### PR DESCRIPTION
**Pull request by zoepage**

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -25,124 +25,333 @@
...
+				</p>
+			</div>
+		</section>
+		<section class="grid-wrapper">
 			<article class="grid">
+				👈 This is the box to rule all modules. 👇 
+			</article>
+		</section>
+		<section class="grid-wrapper">
 				<div class="element x3">
```
Comment (kotzendekrabbe) [inline docs/index.html]: style (indent)

File: docs/index.html [hunk 2]
```diff
@@ -25,124 +25,333 @@
...
+					<pre>
+						<code class="code-text">
+&lt;section class="grid-wrapper"&gt;
+	&lt;article class="grid"&gt;
+	&lt;/article&gt;
+&lt;/section&gt;
+						</code>
+					</pre>
+				</div>
+			</div>
```
Comment (kotzendekrabbe) [inline docs/index.html]: one closing div to much

File: docs/index.html [hunk 3]
```diff
@@ -25,124 +25,333 @@
...
+&lt;article class="grid"&gt;
+	&lt;div class="element x3"&gt;
+		&lt;h2&gt;Title&lt;/h2&gt;
+		&lt;p&gt;Text&lt;/p&gt;
+	&lt;/div&gt;
+&lt;/article&gt;
+						</code>
+					</pre>
+				</div>
+			</div>
```
Comment (kotzendekrabbe) [inline docs/index.html]: one closing div to much

File: docs/index.html [hunk 4]
```diff
@@ -25,124 +25,333 @@
...
-						Mozilla Web Compatibility team, this initiative is a global task force on a mission to improve
-						the web one bug at a time. Join us! Let's work together to make the web a better place to work
-						and play. Learn more about how to contribute.
+						Cupcake ipsum dolor sit amet liquorice lollipop apple pie. Tiramisu pudding pastry chocolate bar icing. Donut sweet candy canes liquorice pie icing. Cookie ice cream sweet roll jujubes carrot cake dessert bear claw. Carrot cake fruitcake soufflé jelly beans jelly beans powder. Halvah cookie croissant jelly beans gummi bears soufflé cake tart jelly-o. Cheesecake cake chocolate chocolate gummi bears lemon drops.
 					</p>
 				</div>
 			</article>
 		</section>
+		<section class="grid-wrapper">
+				<div class="element x3">
```
Comment (kotzendekrabbe) [inline docs/index.html]: indent

File: docs/index.html [hunk 5]
```diff
@@ -25,124 +25,333 @@
...
+	&lt;article class="element x2"&gt;
+		&lt;h2 class="headline-1"&gt;Main part&lt;/h2&gt;
+		&lt;p&gt;Text&lt;/p&gt;
 	&lt;/article&gt;
-&lt;/section&gt;
+&lt;/div&gt;
 						</code>
 					</pre>
 				</div>
 			</div>
```
Comment (kotzendekrabbe) [inline docs/index.html:178]: closing div to much

File: docs/index.html [hunk 6]
```diff
@@ -25,124 +25,333 @@
...
+	&lt;/aside&gt;
+	&lt;article class="element x2"&gt;
+		&lt;h2&gt;Main&lt;/h2&gt;
+		&lt;p&gt;Text&lt;/p&gt;
+	&lt;/article&gt;
+&lt;/div&gt;
+						</code>
+					</pre>
+				</div>
+			</div>
```
Comment (kotzendekrabbe) [inline docs/index.html]: to much closing divs

File: docs/index.html [hunk 7]
```diff
@@ -202,13 +405,15 @@ <h4 class="headline-3">Module: Tiles</h4>
 		</section>
 
 		<section class="grid-wrapper">
-			<div class="element x3 hint-text">
-				<h4 class="headline-3">Button</h4>
-				<p class="note">
-					Button has 4 different shapes: button-primary, button-secondary, button-hero, button-icon<br/>
-				</p>
-			</div>
+				<div class="element x3 hint-text">
```
Comment (kotzendekrabbe) [inline docs/index.html]: indent

File: docs/index.html (hunk 1 above)
Comment (zoepage) [reply docs/index.html]: done.

File: docs/index.html (hunk 5 above)
Comment (zoepage) [reply docs/index.html:178]: done

**Pull request merged**